improver/nowcasting/optical_flow.py

```diff
@@ -265,9 +269,15 @@ def __init__(self, data_smoothing_method="box", iterations=100):
                 Smoothing method to be used on input fields before estimating
                 partial derivatives.  Can be square 'box' (as used in STEPS) or
                 circular 'kernel' (used in post-calculation smoothing).
+            data_smoothing_radius (float):
+                The radius, in km, of the kernel used to smooth the input data fields
+                before calculating optical flow.  14 km is suitable for precipitation
+                rate data separated by a 15 minute time step.  If the time step is
+                greater than 15 minutes, this radius is increased by the "process"
+                method.
```

Comment:
I presume this is only true for data on a 2km grid? If so, you could specify it by making your statement about 14km something like:
```For data on a 2km grid a 14 km radius is suitable for precipitation rate...``` etc

Reply:
No, the reason this is set to 14 km (rather than 7 grid squares) is because it ought to generalise to different grid lengths.  It's about the absolute scale and velocity (in m/s or km/h) of precipitation features.